test/depots/depotDonneesUtilisateurs.spec.js

```diff
@@ -640,4 +641,64 @@ describe('Le dépôt de données des utilisateurs', () => {
         .catch(done);
     });
   });
+
+  describe('sur demande de vérification du mot de passe', () => {
```

Comment:
Passons tout ce `describe` en `async/await`.

Cherche `expect().to.fail(` dans le code pour trouver une inspiration pour le test d'exception.